test/metadata.test.ts

```diff
@@ -2,7 +2,7 @@ import { Connection } from '../src';
 import { MASTER_EDITION_PUBKEY, METADATA_PUBKEY, NETWORK, STORE_OWNER_PUBKEY } from './utils';
 import { Metadata, MetadataKey, MasterEdition } from '@metaplex-foundation/mpl-token-metadata';
 
-describe('Metadata', () => {
+describe.skip('Metadata', () => {
```

Comment:
I don't understand the reasoning here.
How should a test become outdated if you change nothing about the SDK code itself?
At any rate just skipping without an explanation isn't a good solution. Ideally the test would be adapted and thus at the very least you should add a `TODO` here that in a subsequent PR this test will be fixed and re-enabled.

If it is entirely outdated then it should be removed eventually, but also then you'd need to figure out why it is outdated and if you can adapt it instead of removing it entirely.

Reply:
The test failed every time, and  it runs in 'devnet'. There was an idea to rewrite it against local solana, but in this way the test will be the identical as  [createMetadataAndME test]( https://github.com/metaplex-foundation/js/blob/main/test/actions/createMetadataAndME.test.ts), so we have to create metadata and master edition in the both tests to verify them. So I think it's quite unnecessary actions  to duplicate tests, and I'm going to remove on the next iteration

Reply:
If the test is an actual duplicate then it should be removed eventually (not skipped).
This could be done in a separate PR that also points to the duplicate test explaining how they cover the same use case.

Reply:
I absolutely agree, this will be removed in a separate PR